src/components/message/message-card/raw/MessageRaw.tsx

```diff
@@ -19,14 +19,51 @@ import { observer } from 'mobx-react-lite';
 import { useMessagesWorkspaceStore } from '../../../../hooks';
 import DetailedMessageRaw from './DetailedMessageRaw';
 import SimpleMessageRaw from './SimpleMessageRaw';
+import { MessageViewType } from '../../../../models/EventMessage';
 
 interface Props {
 	rawContent: string;
 	messageId: string;
 	renderInfo: () => React.ReactNode;
+	isEmbedded?: boolean;
+	viewType: MessageViewType.ASCII | MessageViewType.BINARY;
 }
 
-function MessageRaw({ rawContent, messageId, renderInfo }: Props) {
+interface EmbeddedProps {
```

Comment:
we can get that prop `isDetailed` in MessageCard and pass down so there would be no need to create a separate component